### DISCUSSION TITLE
DatePicker Display Format

### DISCUSSION THREAD

**Discussion by anilaeus** [filamentphp/filament#11567] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

DatePicker::make('reservation_date')
                                ->label('Rezervasyon Tarihi')
                                ->live(onBlur: true)

I went to DatePicker component inside and   add:
` public static string $defaultDateDisplayFormat = "d/m/Y";`

However its still showing as "mm/dd/YYYY"  on input before selection and after.

Is there a solution for me to show it as "dd/mm/YYYY" .

I need help <3

**Comment by zeeshantariq08** [comment, marked as answer]
You can use this code, i have implemented in my project like this.
```
  Forms\Components\DatePicker::make('expected_implementation_date')
                                  ->displayFormat('d-m-Y')
                                  ->placeholder('dd-mm-yyyy')
                                  ->native(false),`
```

**Reply by anilaeus** [reply]
Thank you for your answer, i just change computer and it suddenly start to work. Maybe its so smart to understand computer local language by default and work for it. I am satisfied with this default behaviour. 

But thank you for your answer anyways
